kaolin/ops/spc/spc.py

```diff
@@ -0,0 +1,316 @@
+# Copyright (c) 2021 NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+__all__ = [
+    'feature_grids_to_spc',
+    'scan_octrees',
+    'generate_points',
+    'to_dense',
+    'query',
+    'points_to_morton',
+    'morton_to_points',
+    'points_to_corners',
+    'points_to_coeffs',
+    'points_to_octree'
+]
+
+import math
+from torch.autograd import Function
+import torch
+
+from kaolin import _C
+from .uint8 import bits_to_uint8
+from kaolin.rep import Spc
+
+def points_to_octree(points, level, sorted=False):
+    r"""Convert points to an octree.
+
+    Args:
+        points (torch.ShortTensor):
```

Comment:
@tovacinni please add some details about what those `points` are, make sure to show the distinction with spc `points`. (they are at a specific level). Add an `unbatched_` prefix. I think there should be a `points = points.contiguous()` at the very beginning of the function?